server/settings/conversion.py

```diff
@@ -0,0 +1,24 @@
+from typing import Any
+
+
+def _convert_validate_subset_name(overrides: dict[str, Any]) -> None:
+    # Convert old "ValidateSubsetName" to new "ValidateProductName"
+    if "publish" not in overrides:
+        return
+
+    publish_overrides = overrides["publish"]
+    if (
+            "ValidateSubsetName" in publish_overrides
```

Comment:
Was the plugin actually renamed in settings model?

Reply:
Well spotted! I may have missed a commit. Let me check.

Reply:
Fixed with https://github.com/ynput/ayon-houdini/pull/245/commits/9c59071812fb92bb4c771335113347da3715eede